project_task_analytic_tag/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Project Task Analytic Tag
+=========================
+
+This module extends the functionality of project and mrp_repair to support to
```

Comment:
Hi @elicoidal , you are right about README, it's a mistake about copy & paste from other module.

But this module is not overlapping with project category. The target to add analytic tags to project task is to create analytic lines (timesheet lines) with right tag.

This way you can set tags like "Design", "Develop", "Testing"... on your timesheet lines, and you will be able to get analytic about type of work you was doing. Also interesting in combination with analytic dimensions.